min-turnup/phase2/ansible/playbooks/roles/master/templates/kubeconfig.jsonnet

```diff
@@ -1,26 +0,0 @@
-function(cfg)
```

Comment:
isn't this required to create /srv/kubernetes/kubeconfig.json? I don't see what it was replaced by.